Dockerfile

```diff
@@ -1,29 +1,12 @@
 # syntax = docker/dockerfile:1.2
 
 FROM rust:1.76-bullseye as builder
```

Comment:
This line seems not longer needed.

Reply:
:+1: I need to revert all the build changes actually; they were accidentally committed.